lib/parsers/package-lock-parser.ts

```diff
@@ -261,11 +294,18 @@ export class PackageLockParser implements LockfileParser {
         const subDep = depTrees[subDepPath];
         dep.dependencies[subDep.name] = subDep;
       }
-      delete dep.requires;
-      depTrees[depKey] = {...dep as PkgTree};
+      const pkgTree: PkgTree = {
+        cyclic: dep.cyclic,
```

Comment:
FYI I see tests fail as `cyclic` & `hasDevDependencies` are always set although they are optional and sometimes not defined on `dep`.